bundles/org.eclipse.passage.ldc.pde.ui.templates/.settings/.api_filters

```diff
@@ -3,7 +3,7 @@
     <resource path="META-INF/MANIFEST.MF">
         <filter comment="Bulk version increment" id="925892614">
             <message_arguments>
-                <message_argument value="2.8.0"/>
```

Comment:
I would rather expect `.api_filters` deleted later as a result of baseline switch.
For this PR let's revert changes for `.api_filters` if we don't touch baseline.